pkg/apis/core/validation/validation.go

```diff
@@ -4421,11 +4422,11 @@ func validateServiceExternalTrafficFieldsValue(service *core.Service) field.Erro
 	return allErrs
 }
 
-// ValidateServiceExternalTrafficFieldsCombination validates if ExternalTrafficPolicy,
+// validateServiceExternalTrafficFieldsCombination validates if ExternalTrafficPolicy,
```

Comment:
might worth a todo to separate the tests against Create/Update that focuses on ExternalTrafficPolicy into a separate function. The test Create/Update funcs are just too big to comprehend.

Reply:
Agree - lots of room for cleanup